src/threading/MsgThread.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <atomic>
+#include <memory>
```

Comment:
Since you are at it, many of these are unused, and we only seem to need these includes:

```cpp
#include <memory>

#include "zeek/threading/BasicThread.h"
#include "zeek/threading/Queue.h"
```

Reply:
DebugLogger.h is needed for `void Debug(DebugStream stream, const char* msg);` but the rest can go.